CMakeLists.txt

```diff
@@ -19,9 +19,19 @@ set(SUPERBUILD_TOPLEVEL_PROJECT inner)
 #-----------------------------------------------------------------------------
 # Extension dependencies
 find_package(Slicer REQUIRED)
+# Skip implicit generation of launcher settings triggered by SlicerBlockAdditionalLauncherSettings
+# It was generating the CMake Warning "Dependent extension SlicerIGT cannot be found by CMake find_package() ..."
+set(Slicer_SKIP_SlicerBlockAdditionalLauncherSettings TRUE)
```

Comment:
Instead of setting it to a hardcoded `TRUE`, could we set this externally for the special case when SlicerHeart is built without SlicerIGT? Or, could we make EXTENSION_DEPENDS setting conditional, based on what modules are enabled/disabled?